src/main/java/seedu/address/logic/commands/ShowChildrenListCommand.java

```diff
@@ -12,12 +12,25 @@
  * Show Task List.
  */
 public class ShowChildrenListCommand extends Command {
+
     public static final String COMMAND_WORD = "ls";
+
     public static final String MESSAGE_SUCCESS = "Show children List of %1$s";
+
     public static final String MESSAGE_PATH_NOT_FOUND = "Path does not exist in ProfBook: %1$s";
+
     public static final String MESSAGE_NOT_CHILDREN_MANAGER = "Cannot show children list for this path: %1$s";
-    public static final String MESSAGE_USAGE = COMMAND_WORD;
+
+    public static final String MESSAGE_USAGE =
+            "Usage: " + COMMAND_WORD + "\n"
+            + "\n"
+            + "Display the children list.\n"
+            + "\n"
+            + "Examples: \n"
+            + "ls";
```

Comment:
ls can take in optional argument path
e.g. ls grp-001
if not path provided will default to current path